fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -2492,15 +2478,15 @@ public Map<String, String> getProperties() {
         properties.put(PropertyAnalyzer.PROPERTIES_INMEMORY, isInMemory().toString());
 
         Map<String, String> tableProperty = getTableProperty().getProperties();
-        if (tableProperty != null) {
-            if (tableProperty.containsKey(PropertyAnalyzer.PROPERTIES_STORAGE_MEDIUM)) {
-                properties.put(PropertyAnalyzer.PROPERTIES_STORAGE_MEDIUM,
-                        tableProperty.get(PropertyAnalyzer.PROPERTIES_STORAGE_MEDIUM));
-            }
-            if (tableProperty.containsKey(PropertyAnalyzer.PROPERTIES_STORAGE_VOLUME)) {
-                properties.put(PropertyAnalyzer.PROPERTIES_STORAGE_VOLUME,
-                        tableProperty.get(PropertyAnalyzer.PROPERTIES_STORAGE_VOLUME));
-            }
+        if (tableProperty != null && tableProperty.containsKey(PropertyAnalyzer.PROPERTIES_STORAGE_MEDIUM)) {
+            properties.put(PropertyAnalyzer.PROPERTIES_STORAGE_MEDIUM,
+                    tableProperty.get(PropertyAnalyzer.PROPERTIES_STORAGE_MEDIUM));
+        }
+        StorageVolumeMgr svm = GlobalStateMgr.getCurrentState().getStorageVolumeMgr();
+        String storageVolumeId = svm.getStorageVolumeIdOfTable(id);
+        if (storageVolumeId != null) {
+            String volume = svm.getStorageVolume(storageVolumeId).getName();
```

Comment:
better get volume name by table id directly in manager, because name can be modified and should be got with lock.